cabot/settings.py

```diff
@@ -16,6 +16,8 @@
 
 MANAGERS = ADMINS
 
+DEFAULT_FROM_EMAIL = os.environ.get('CABOT_FROM_EMAIL')
```

Comment:
Do we want to risk this being `None` in the case that `CABOT_FROM_EMAIL` isn't set? Should we provide a default (`webmaster@localhost`) or something?

I'm keen to avoid the risk of breaking existing installs.